src/main/java/uk/gov/companieshouse/chipsrestinterfacesconsumer/service/impl/MessageProcessorServiceImpl.java

```diff
@@ -37,33 +38,42 @@
     @Autowired
     private MessageProducer messageProducer;
 
+    @Autowired
+    private ConsumerConfig consumerConfig;
+
     @Override
-    public void processMessage(ChipsRestInterfacesSend message) throws ServiceException {
+    public void processMessage(ChipsRestInterfacesSend message, boolean isErrorAttempt) throws ServiceException {
         Map<String, Object> logMap = new HashMap<>();
         logMap.put("Message", message.getData());
         try {
             chipsRestClient.sendToChips(message);
         } catch (HttpStatusCodeException hsce) {
             logMap.put("HTTP Status Code", hsce.getStatusCode().toString());
-            handleFailedMessage(message, hsce, logMap);
+            handleFailedMessage(message, hsce, logMap, isErrorAttempt);
         } catch (Exception e) {
-            handleFailedMessage(message, e, logMap);
+            handleFailedMessage(message, e, logMap, isErrorAttempt);
         }
     }
 
-    private void handleFailedMessage(ChipsRestInterfacesSend message, Exception e, Map<String, Object> logMap) throws ServiceException {
+    private void handleFailedMessage(ChipsRestInterfacesSend message, Exception e, Map<String, Object> logMap, boolean isErrorAttempt) throws ServiceException {
         var messageId = message.getMessageId();
         logger.error(String.format(SEND_FAILURE_MESSAGE, messageId), e, logMap);
 
+        if (isErrorAttempt) {
+            message.setAttempt(1);
```

Comment:
could we set attempt to 1 also when processing message for first time on main? to ensure they all start from the same attempt number.

Reply:
not easily enough that I think it should be done in this pr